src/GraphQL/Execution/ParallelExecutionStrategy.cs

```diff
@@ -6,7 +6,10 @@ namespace GraphQL.Execution
 {
     public class ParallelExecutionStrategy : ExecutionStrategy
     {
-        protected override async Task ExecuteNodeTreeAsync(ExecutionContext context, ObjectExecutionNode rootNode)
+        protected override Task ExecuteNodeTreeAsync(ExecutionContext context, ObjectExecutionNode rootNode)
```

Comment:
Is there a reason why this Async method is not awaitable?

Reply:
It sure is awaitable. It just forwards to the implementation below which accepts an `ExecutionNode` instead of an `ObjectExecutionNode`.

Since there is no `await` used in forwarding to the other method, you don`t need to use the `async` keyword. You just return the async Task created by the forwarded method.